api/migrations/0002_events.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
@NinaNinchy this file does not need to be inhere just yet. We will make one migration file before we put everything into production, so we can have things a bit more tidied up.